variables.tf

```diff
@@ -897,6 +897,19 @@ variable "msi_auth_for_monitoring_enabled" {
   description = "(Optional) Is managed identity authentication for monitoring enabled?"
 }
 
+variable "nat_gateway_profile" {
+  type = object({
+    idle_timeout_in_minutes   = optional(number)
+    managed_outbound_ip_count = optional(number)
+  })
+  default     = null
+  description = <<-EOT
+ `nat_gateway_profile` block supports the following:
+ - `idle_timeout_in_minutes` - (Optional) Desired outbound flow idle timeout in minutes for the cluster load balancer. Must be between `4` and `120` inclusive. Defaults to `4`.
```

Comment:
Doing a review of this PR I found out that the upstream provider documentation was wrong. Also for the parameters.

I opened a PR here:
https://github.com/hashicorp/terraform-provider-azurerm/pull/27287/files